tests/neptune/internal/backends/test_noop_object.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+from neptune.internal.backends.offline_backend import NoopObject
+
+class TestHostedNeptuneObject(unittest.TestCase):
+
+    # given
+    objectUnderTest = NoopObject()
```

Comment:
While it's true that this object should not be mutable, I think it would be prudent to keep it scoped to a single test.
Right now, tests cases can interact with each other.